tinywallet/tinywallet/screens.py

```diff
@@ -1975,9 +1991,9 @@ def setStatsNetwork(self):
         self.blkHeight.setText(str(tpinfo.height))
         self.networkTickets.setText(t(tpinfo.size, ", "))
         self.networkValue.setText("{:.2f} dcr".format(tpinfo.value))
-        blocksLeft = calc.blksLeftStakeWindow(cfg.netParams, tpinfo.height)
+        blocksLeft = blksLeftStakeWindow(tpinfo.height, cfg.netParams)
         self.blocksLeft.setText(b(blocksLeft))
-        cache = calc.SubsidyCache(cfg.netParams)
+        cache = txscript.SubsidyCache(cfg.netParams)
```

Comment:
Here's the cache that I think can go. The `blockchain` = `DcrdataBlockchain` has a `SubsidyCache`. 

`cache = blockchain.subsidyCache`